CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ endif()
 
 # Our output dir
 set(CMAKE_BINARY_DIR ${PROJECT_BINARY_DIR})
+#set(CMAKE_INCLUDE_CURRENT_DIR_IN_INTERFACE ON) # unfortunately, serializes everything
```

Comment:
what does 'serializes everything' mean?

Reply:
"Makes it so it compiles libraries in series as opposed to in parallel" is what I meant